libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -624,39 +624,36 @@ - (void)setOauthClientId:(NSString *)newClientId
 
 + (void)resetSessionCookie
 {
-    [self removeCookies:@[@"sid"]
-            fromDomains:@[@".salesforce.com", @".force.com", @".cloudforce.com"]];
-    [self addSidCookieForInstance];
+    __weak typeof(self) weakSelf = self;
```

Comment:
Yeah, but all of this cookie cleanup applies to both auth and hybrid views.

It seems like, for the methods we keep on this front, they need to clean up both the `NSHTTPCookieStorage` data  _and_ the `WKProcessPool` instance—I'd posit that we should get out of the business of cleaning up the `WKWebsiteDataStore` altogether for the WK side, in favor of simply clearing the `WKProcessPool` that we share.